rpa-bot/1.3/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -28,12 +30,28 @@ include::rpa-home::partial$windows-versions-support.adoc[]
 
 |===
 
-=== Requirements for Secure Session Technology
+=== System Settings
+
+* Display resolution: 
+** Minimum: 1280 x 1080
+** Recommended: Full HD 1080p (1920 × 1080) or higher
+* Scaling of text and other items on the screen: 100 %
+* _Clear Type_: turned off
+
+=== Requirements for the Secure Session Technology
+
+Secure Sessions are hidden Desktop sessions. They run in the background and only the bot can access them. An additional advantage of using Secure Sessions is, that the bot can multiply itsself to run several process instances in parallel.
```

Comment:
```suggestion
Secure Sessions are hidden Desktop sessions that run in the background and are accessible only to the bot. An additional advantage of using Secure Sessions is that the bot can multiply itself to run several process instances in parallel.
```

Reply:
I'm breaking the cardinal rue of active voice here :(

Reply:
You mention an additional advantage, but I don't see the original advantage.

This paragraph is really about what Secure Sessions are and the advantages, rather than requirements.

Reply:
No problem for me. I like passive voice :)

Reply:
Yes, I included the explanation of what Secure Sessions are because the problem seems to be that our users can't seem to find that information easily.

1st advantage: hidden
2nd advantage: parallel

Reply:
Ah, I missed that hidden=advantage :>